package/MDAnalysis/coordinates/INPCRD.py

```diff
@@ -22,29 +22,92 @@
 #
 
 
-"""INPCRD structure files in MDAnalysis --- :mod:`MDAnalysis.coordinates.INPCRD`
+"""AMBER restart files in MDAnalysis --- :mod:`MDAnalysis.coordinates.INPCRD`
 ================================================================================
 
-Read coordinates in Amber_ coordinate/restart file (suffix "inpcrd").
+AMBER_ can write :ref:`ASCII restart<ascii-restart>` ("inpcrd") and
+:ref:`binary restart<netcdf-restart>` ("ncrst") coordinate files. MDAnalysis
+supports reading of both file formats.
 
-.. _Amber: https://ambermd.org/FileFormats.php
+.. rubric:: Units
 
+AMBER restart files are assumed to be in the following units:
 
-Classes
--------
+* length in Angstrom (Å)
+* time in ps
+* velocity (NCRST only) in Å / ps
+* force (NCRST only) in kcal / (mol * Å)
+
+
+.. _ascii-restart:
+
+ASCII INPCRD restart files
+--------------------------
+
+ASCII AMBER_ INPCRD coordinate files (as defined in `AMBER INPCRD FORMAT`_)
+are handled by the :class:`INPReader`.
+
+AMBER ASICC restart files are recognised by the suffix '.inpcrd', '.restrt', or
+'.rst7'
 
 .. autoclass:: INPReader
    :members:
 
+
+.. _netcdf-restart:
+
+Binary NetCDF restart files
+---------------------------
+
+The `AMBER netcdf`_ restart format makes use of NetCDF_ (Network Common Data
+Form) format. Such binary restart files are recognised in MDAnalysis by the
+suffix '.ncrst', '.ncrestrt' or '.ncrst7' and read by the :class:`NCRSTReader`.
+
+Binary restart files can also contain velocity and force information, and can
+record the simulation time step. Whilst the `AMBER netcdf`_ format details
+default unit values of ångström and picoseconds, these can in theory occupy
+any unit type. However, at the moment MDAnalysis only supports the default
+types and will raise a :exc:`NotImplementedError` if anything else is detected.
+
+.. autoclass:: NCRSTReader
+   :members:
+
+
+.. Links
+
+.. _AMBER: http://ambermd.org
+.. _AMBER INPCRD FORMAT: http://ambermd.org/formats.html#restart
```

Comment:
this link is 404 when I accessed it, maybe this one?

https://ambermd.org/FileFormats.php#restart